lib/git.js

```diff
@@ -16,21 +16,15 @@ module.exports = options => {
 		{
 			title: 'Check local working tree',
 			task: async () => {
-				const {stdout: status} = await execa('git', ['status', '--porcelain']);
-				if (status !== '') {
+				if (!(await isWorkingTreeClean())) {
 					throw new Error('Unclean working tree. Commit or stash changes first.');
 				}
```

Comment:
Should we also move this to `git-util` as `verifyWorkingTreeIsClean()`? Same with the other ones.

Reply:
I think so 👌🏻